parlai/tasks/light_dialog/agents.py

```diff
@@ -86,6 +87,9 @@ def add_cmdline_args(argparser):
             default='all',
             choices=['partner', 'self', 'all', 'none'],
         )
+        agent.add_argument(
+            '--light_use_affordances', type=str, default='all', choices=['all', 'none']
```

Comment:
do you foresee there being more than two options in the future? if not, a more intuitive arg might be a bool `--light-use-all-affordances`.